demo/demo_frame_graphs.hpp

```diff
@@ -219,6 +234,17 @@ namespace fg_manager
 		}
 	}
 
+	inline void UpdateIfTasksChanged(wr::RenderSystem& rs, util::Delegate<void(ImTextureID)> imgui_func)
```

Comment:
Remove this please. I don't want this code in master since we can't test it correctly before the play day. I am not convinced rebuilding the SG works properly.